applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py

```diff
@@ -54,6 +55,8 @@ def AssembleTestSuites():
     smallSuite.addTest(EmbeddedCouetteImposedTest('testEmbeddedCouetteImposed2D'))
     smallSuite.addTest(EmbeddedReservoirTest('testEmbeddedReservoir2D'))
     smallSuite.addTest(EmbeddedReservoirTest('testEmbeddedSlipReservoir2D'))
+    smallSuite.addTest(EmbeddedVelocityInletEmulationTest('testEmbeddedVelocityInletEmulationEmbedded2D'))
```

Comment:
If you do it like [this](https://github.com/KratosMultiphysics/Kratos/blob/bab2ce374ab0ba1813e83fd0cdc8f453631d2e22/applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py#L370) then it automatically adds all the tests in the test-class
Handy when you add more tests later, as they will automatically be added

Reply:
Good to know. Thanks @philbucher .